packages/atomic/src/components/facets/facet-segmented-value/facet-segmented-value.pcss

```diff
@@ -0,0 +1,24 @@
+.box-container {
```

Comment:
we should be able to delete most of what is in that file now that we're using a different functional component